solveengine/mipmodel.py

```diff
@@ -220,7 +254,7 @@ def set_to_maximize(self):
         self.set_direction(Direction.MAXIMIZE)
     
     def build_with_matrices(self, f, A, b, 
-                            Aeq = None, beq = [], 
+                            Aeq=None, beq=[],
                             lb=[], ub=[], 
                             int_list=[], bin_list=[]):
```

Comment:
lists are unsafe as default values because they are mutable, us tuples instead
i.e. this code
```
def t(a=[]):
    a.extend([1])
    print(a)
t()
t()
```
prints
```
[1]
[1, 1]
```